### PR TITLE
build: Initialize changesets

### PR DESCRIPTION
**Pull request by tylerbutler**
This PR initializes changesets in the client and azure release groups. This only adds the dependencies and config files. CI enforcement and tool improvements will be in future PRs.

### CODE REVIEW

File: azure/.changeset/config.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"$schema": "https://unpkg.com/@changesets/config@2.3.0/schema.json",
+	"changelog": [
+		"changesets-format-with-issue-links",
+		{
+			"repoBaseUrl": "https://github.com/microsoft/FluidFramework",
```
Comment (Josmithr) [inline azure/.changeset/config.json:6]: Sanity check: what is this used for? Should this point to the azure dir?

File: .changeset/config.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
+{
```
Comment (Josmithr) [inline .changeset/config.json:1]: Are these configs extensible? Would it be possible and useful to add a base config in a shared location each release group can extend?

File: .changeset/README.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Changesets
+
+This folder contains changesets, which are markdown files that hold two key bits of information: a
+version type (following semver), and change information to be added to a changelog. You can find the full documentation
```
Comment (Josmithr) [inline .changeset/README.md:4]: Nit: I think this would be a bit easier to read if the 2 elements listed here were formatted as a Markdown list.

**Review by Josmithr — APPROVED** [review]
It would probably be worth updating the 2 release group READMEs to document this tooling and how it is intended to be used.

Also, does the changeset tooling support comments in the `config.json` files? If so, some brief comments explaining what some of the properties mean / how they are used might be helpful for folks making changes in the future. The `fixed` property in particular is non-obvious.

**Comment by tylerbutler** [comment]
> It would probably be worth updating the 2 release group READMEs to document this tooling and how it is intended to be used.

That'll be in a later change.

> does the changeset tooling support comments in the config.json files?

Alas, they do not. I filed https://github.com/changesets/action/issues/277 though.

File: .changeset/config.json (hunk 2 above)
Comment (tylerbutler) [reply .changeset/config.json:1]: Unfortunately, they're not extensible as far as I can tell.

File: azure/.changeset/config.json (hunk 1 above)
Comment (tylerbutler) [reply azure/.changeset/config.json:6]: It's used to generate links to issues and PRs in the changelog, so this is correct.

**Pull request merged**